esmvaltool/cmorizers/data/downloaders/datasets/cfsr.py

```diff
@@ -0,0 +1,92 @@
+"""Script to download CFSR from RDA."""
+import logging
+import os
+
+from datetime import datetime
+
+from dateutil import relativedelta
+
+from esmvaltool.cmorizers.data.downloaders.wget import WGetDownloader
+from esmvaltool.cmorizers.data.utilities import unpack_files_in_folder
+
+
+logger = logging.getLogger(__name__)
+
+
+def download_dataset(config, dataset, dataset_info, start_date, end_date,
+                     overwrite):
+    """Download dataset.
+
+    Parameters
+    ----------
+    config : dict
+        ESMValTool's user configuration
+    dataset : str
+        Name of the dataset
+    dataset_info : dict
+         Dataset information from the datasets.yml file
+    start_date : datetime
+        Start of the interval to download
+    end_date : datetime
+        End of the interval to download
+    overwrite : bool
+        Overwrite already downloaded files
+    """
+    downloader = WGetDownloader(
+        config=config,
+        dataset=dataset,
+        dataset_info=dataset_info,
+        overwrite=overwrite,
+    )
+
+    os.makedirs(downloader.local_folder, exist_ok=True)
+
+    user = os.environ.get("rda-user")
+    if user is None:
+        user = str(input("RDA user name? "))
+        if user == "":
+            errmsg = ("A RDA account is required to download JRA-55 data."
+                      " Please visit https://rda.ucar.edu/login/register/"
+                      " to create an account at the Research Data Archive"
+                      " (RDA) if needed.")
+            logger.error(errmsg)
+            raise ValueError
+
+    passwd = os.environ.get("rda-passwd")
+    if passwd is None:
+        passwd = str(input("RDA password? "))
+
+    if start_date is None:
+        start_date = datetime(1979, 1, 1)
+    if end_date is None:
+        end_date = datetime(1979, 12, 31)
```

Comment:
The period should be  until 2010
```suggestion
        end_date = datetime(2010, 12, 31)
```